config/locales/mfa.en.yml

```diff
@@ -13,7 +13,7 @@ en:
       phone:
         nochange: Your account is already using this mobile number. Enter a different number.
       phone_code:
-        expired: The security code you’ve entered has expired. Security codes expire after 10 minutes. We can <a class="govuk-link" href="%{resend_link}">send a new security code</a>.
+        expired: The security code you’ve entered has expired. Security codes expire after 30 minutes. We can <a class="govuk-link" href="%{resend_link}">send a new security code</a>.
```

Comment:
Not essential, but a possible improvement: can we get the time from the config?  Then we don't need remember to update the content if the time changes in the future.

Reply:
Yeah would be happy to look at that, but for now this message is never shown to users.
I've only really left it in place rather than delete because there's some follow up work i suspect will see it return (perhaps with a new value) qutie soon?